### DISCUSSION TITLE
IP pool capability question

### DISCUSSION THREAD

**Discussion by PrabhuShanmughapriyan** [postalserver/postal#2745] [Help with using Postal]
HI,

I want to scale number of workers in different machines for HA. 

I have the following setup:

1 main machine with all of postal including DB , RabbitMQ etc.
2nd machine with only workers pointing to DB / Queue in machine 1.

It is working fine, but there is no control of which server 1 or 2 picks the email. As of now it is working fine. So there is no need for me to configure IP pool.. Am I right?

Question 1
**Is IP pool only for multiple IPs attached to the same machine?**

Question 2
 **How can we do _domain1.example.com_ to send emails for _IP1_ and _domain2.example.com_ to send emails from _IP2_ which is running worker alone on a different machine**

**Comment by willpower232** [comment]
IP Pools were intended for using specific IP addresses on the machine sending out messages, I'm not sure what would happen if you set up the IP Pools so that domain1 could only be sent from one machine and domain2 could only be sent from the other. If you give it a go and see what happens, I think the workers are smart enough to pick up messages they can send from IPs they are connected to.
